src/microphysics/eos_phoebus/eos_phoebus.hpp

```diff
@@ -17,11 +17,27 @@
 #include <memory>
 #include <parthenon/package.hpp>
 
+#include <singularity-eos/eos/eos.hpp>
+
 using namespace parthenon::package::prelude;
 
 namespace Microphysics {
 
+//  using MyEOS=singularity::impl::Variant<IdealGas>;
+
 namespace EOS {
+
+#ifdef SPINER_USE_HDF
+using UnmodifiedEOS =
+    singularity::Variant<singularity::IdealGas, singularity::SpinerEOSDependsRhoT,
+                         singularity::SpinerEOSDependsRhoSie,
+                         singularity::StellarCollapse>;
+#else
+using UnmodifiedEOS = singularity::Variant<singularity::IdealGas>;
+#endif
+
+using EOS = UnmodifiedEOS;
```

Comment:
Our variant should actually be the `UnitSystem<IdealGas>` for example. We can also probably throw away the `SpinerEOSDepends...` ones.

Reply:
Ah right thanks

Reply:
Done